tools/DfmHttpService/Program.cs

```diff
@@ -19,7 +19,7 @@ public static void Main(string[] args)
                     new ExitHandler()
                 });
 
-            var service = new DfmHttpServer(handler, args.Length > 0 ? args[0] : null);
+            var service = new DfmHttpServer(handler, "localhost", args.Length > 0 ? args[0] : null);
```

Comment:
DfmHttpServer is the background server used by vscode extension. Let's keep it "localhost" for now.